packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/FGMClusterManagersController.h

```diff
@@ -0,0 +1,70 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Flutter/Flutter.h>
+#import <GoogleMaps/GoogleMaps.h>
+@import GoogleMapsUtils;
+
+#import "messages.g.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+// Defines cluster managers controller interface which
```

Comment:
Comments should be written for an audience that understand the basic mechanics of the language; a class comment does not need to say that `@interface` defines an interface.

Also, this should be a `///` comment.

```objc
/// A controller that manages all of the cluster managers on a map.
```